sdk/core/core-amqp/src/requestResponseLink.ts

```diff
@@ -249,17 +232,8 @@ export class RequestResponseLink implements ReqResLink {
         );
         this.sender.send(request);
       });
-    const config: RetryConfig<AmqpMessage> = {
-      operation: sendRequestPromise,
-      connectionId: this.connection.id,
-      operationType:
-        request.to && request.to === Constants.cbsEndpoint
-          ? RetryOperationType.cbsAuth
-          : RetryOperationType.management,
-      delayInSeconds: options.delayInSeconds,
-      maxRetries: options.maxRetries
-    };
-    return retry<AmqpMessage>(config);
+
+    return sendRequestPromise();
```

Comment:
`sendRequestPromise` was a function previously because we had to pass a function that returns a promise to the `retry()`  before. Since that is no longer needed, we don't need `sendRequestPromise` to be a function.
It can be a constant or you can return the promise from its line of creation i.e from `new Promise(...)`